static/js/tools/timeline_util.js

```diff
@@ -170,6 +170,28 @@ function getStatsVarInfo(dcids) {
   });
 }
 
+function getStatsVar(dcids) {
```

Comment:
can you make this file a typescript file and add typing.

Reply:
done.